ppr-ui/src/views/amendment/ConfirmAmendment.vue

```diff
@@ -382,6 +382,11 @@ export default class ConfirmAmendment extends Vue {
   private get courtOrderValid (): boolean {
     const courtOrder: CourtOrderIF = this.getCourtOrderInformation
     return (!courtOrder ||
+            (courtOrder.courtName.length === 0 &&
+            courtOrder.courtRegistry.length === 0 &&
+            courtOrder.fileNumber.length === 0 &&
+            courtOrder.orderDate.length === 0 &&
+            courtOrder.effectOfOrder.length === 0) ||
```

Comment:
Hey Doug can you just have the last conditional as >= instead here?

Reply:
Kial, all court order data elements are mandatory: a character length of 0 will fail API validation and a database insert constraint.

Reply:
ah I see sorry -- I was thinking it would be the same check, but I see you're validating that every value is empty or every value has something